CHANGELOG.md

```diff
@@ -19,6 +19,8 @@
 
 #### Core
 
+- Add `entity_checks` to /schemas endpoint output
```

Comment:
```suggestion
- '/schemas' endpoint returns additional information about cross-field validation as part of the schema. This should help tools that use the Admin API to perform better client-side validation. 
```

A good changelog entry always describes the change from a user's perspective. Once again, the changelog is meant for users and not maintainers.